addons/website_sale/controllers/main.py

```diff
@@ -432,13 +432,15 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, t
 
         ProductAttribute = request.env['product.attribute']
         if products:
-            # get all products without limit
-            attributes = lazy(lambda: ProductAttribute.search([
+            attributes_domain = [
                 ('product_tmpl_ids', 'in', search_product.ids),
                 ('visibility', '=', 'visible'),
-            ]))
+            ]
         else:
-            attributes = lazy(lambda: ProductAttribute.browse(attribute_ids))
+            attributes_domain = [('id', 'in', attribute_ids)]
+        # get all attributes without limit ordered by sequence to
+        # bypass website_sale_comparison setting order to category_id
+        attributes = lazy(lambda: ProductAttribute.search(attributes_domain, order='sequence'))
```

Comment:
Talked about it with Jean, and we believe the category should only impact the ordering on the comparison page, not the generic ordering of the attributes.

Could you please remove the custom `_order` on `product.attribute` model in `website_sale_comparison` instead (and remove the specific order here so that it follows the default order of the model) ?

Thanks :pray: